src/Form.vue

```diff
@@ -98,6 +102,10 @@ export default {
       question.isValid = true;
       question.validationMessage = "";
     },
+    setIsMandatory(question) {
+	question.isMandatory = ["list", "rawlist", "checkbox", "expand"].includes(question.type) ||
```

Comment:
checkbox is not mandatory by default! it can be with empty answer.

Reply:
Agree. Done